src/main/java/net/fabricmc/loader/api/FabricLoader.java

```diff
@@ -154,8 +154,11 @@ static FabricLoader getInstance() {
 	 * server object. As such, the exact return is dependent on the
 	 * current environment type.
 	 *
+	 * <p>The game instance may not always be availible depending on the game version.
+	 *
 	 * @return A client or server instance object
 	 */
+	/* @Nullable */
 	Object getGameInstance();
```

Comment:
Imo we should deprecate this method in the future. some game logic just happen in a static context.